File: ncDados/dCurvaABC.vb

Namespace nsCurvaABC
    Public Class ColecaodVendasABC
        Inherits List(Of dCurvaAbc)
    End Class
    Public Class dCurvaAbc

        Public Property PeriodoIni As String
        Public Property PeriodoFim As String
        Public Property Fabricante As String
        Public Property valor As String
        Public Property PercReceita As Decimal
        Public Property PercAcumulado As Decimal
        Public Property ClasseAbc As String
        Public Property Estrategia As String
    End Class

End Namespace

File: ncModelos/dCurvaABC.cs
using System;
using System.Collections.Generic;

namespace ncModelos
{
    public class ColecaodVendasABC : List<dCurvaAbc>
    {
    }

    public class dCurvaAbc
    {
        public string PeriodoIni { get; set; }
        public string PeriodoFim { get; set; }
        public string Fabricante { get; set; }
        public string valor { get; set; }
        public decimal PercReceita { get; set; }
        public decimal PercAcumulado { get; set; }
        public string ClasseAbc { get; set; }
        public string Estrategia { get; set; }
    }

}
